haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -42,6 +42,8 @@
     "ml": "malayalam",
 }
 
+EMPTY_PAGE_PLACEHOLDER = "[substitut text per ina pagina vida]."
```

Comment:
nit: can we use a more obvious placeholder? Something like `@@@HAYSTACK_KEEP_PAGE@@@`

Reply:
Sure. We should leave the dot though to make sure the sentence tokenizer is splitting up sentences over empty pages.